doc/api/n-api.md

```diff
@@ -4093,7 +4093,7 @@ added: REPLACEME
 ```C
 NAPI_EXTERN napi_status
 napi_get_threadsafe_function_context(napi_threadsafe_function func,
-                                     void** result);
+                                     void** context);
```

Comment:
It's `restuls` and not `context` in the source code (`src/node_api.h` and `src/node_api.cc`). Those should be updated to match as well?

Reply:
If so, please, anybody familiar with C++, push a commit in this PR or suggest a diff. I am afraid to break something)

Reply:
@vsemozhetbyt just keep `result`, it matches with the signature in `src/node_api.h`.

Reply:
Should I replace `context` with `result` then in options below so that the signature matches options?

Reply:
Yes correct.

Reply:
Done.